src/app.js

```diff
@@ -315,36 +341,35 @@ App.prototype.getDefaultOutputElement = function( args ) {
  * @returns {void}
  */
 App.prototype.switchAssessors = function( useCornerStone ) {
```

Comment:
If I understand correctly this function sets the app's assessors based on the `._assessorOptions`, just like `changeAssessorOptions` does but this one is specifically for `userCornerStone` only. I guess this is due to backwards compatibility, but can't we think of a cleaner way to solve this?

Reply:
As discussed, I added `@deprecated` to the documentation of switchAssessors.